tests/src/main/java/net/neoforged/neoforge/debug/client/DimensionTransitionScreenTests.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.debug.client;
+
+import java.util.function.BooleanSupplier;
+import net.minecraft.client.gui.GuiGraphics;
+import net.minecraft.client.gui.screens.ReceivingLevelScreen;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.level.Level;
+import net.neoforged.api.distmarker.Dist;
+import net.neoforged.neoforge.client.event.RegisterDimensionTransitionScreenEvent;
+import net.neoforged.neoforge.event.entity.player.PlayerEvent;
+import net.neoforged.testframework.DynamicTest;
+import net.neoforged.testframework.annotation.ForEachTest;
+import net.neoforged.testframework.annotation.TestHolder;
+import net.neoforged.testframework.gametest.EmptyTemplate;
+
+@ForEachTest(side = Dist.CLIENT, groups = DimensionTransitionScreenTests.GROUP)
```

Comment:
Please put these tests in a `manual` group too (I want to introduce this pattern so that all manual tests are easier to run after a port). Additionally, is there a reason for the tests to be enabled by default? Unless there is one, it should be avoided as it will lead to test overload when you only want to test another test that's not related.